docs/installation.md

```diff
@@ -0,0 +1,24 @@
+# Installation
+
+It is recommended to use a Python virtual environment to reduce the risk of any Python package conflicts.
+
+### HPC Workstation or Local PC
```

Comment:
```suggestion
```



Same as in the README:

> I wouldn't bother saying whether it's for an HPC workstation or local PC since the install instructions are straightforward enough to work for any ordinary python envrionment.